frontend/src/container/NotificationWall.jsx

```diff
@@ -396,7 +406,11 @@ export class NotificationWall extends React.Component {
                 >
                   <span className='notification__list__item__icon'>{icon}</span>
                   <div className='notification__list__item__text'>
-                    <Avatar size={AVATAR_SIZE.MINI} publicName={notification.author} style={{ marginRight: '5px' }} />
+                    <Avatar
+                      size={AVATAR_SIZE.MINI}
+                      publicName={notification.author ? notification.author.publicName : 'unkown'}
```

Comment:
shouldn't unknown be translated ?
If "unknown" is handled specifically, it should be a constant

Reply:
Updated